Sourcery-Graduates/GymAppBE
# Feature/gym 40 be create auditor class

# So, I think I added Audit class for Spring JPA

## How to test:
- go to (http://localhost:8080/swagger-ui/index.html)[http://localhost:8080/swagger-ui/index.html];
- there you will see, the endpoints for permission: get all, create one, and get one by id (may be deleted if not needed, it's for testing puposes)
- try them in postman

Fields createdBy and ModifiedBy are autofilled with UUID: `00000000-0000-0000-0000-000000000001`

-----------------------
# Important:

I disabled the spring security for now in the main application file. So, no blocking by security for now
